ios/RCTFBSDK/share/RCTConvert+FBSDKSharingContent.m

```diff
@@ -112,10 +112,6 @@ static void RCTAppendGenericContent(RCTFBSDKSharingContent contentObject, NSDict
   NSURL *videoURL = [RCTConvert NSURL:videoData[@"localUrl"]];
   FBSDKShareVideo *video = [FBSDKShareVideo videoWithVideoURL:videoURL];
   videoContent.video = video;
-  if (contentData[@"previewPhoto"]) {
```

Comment:
`previewPhoto` still exists but is on `video` now.

Something like:
`videoContent.video. previewPhoto = previewPhoto`

Reply:
nice find. i will create a pr to add it back the correct way.